### PR TITLE
Fix colors in debug logs

### PR DESCRIPTION
**Pull request by lorefnon**
Resolves https://github.com/knex/knex/issues/3588

### CODE REVIEW

File: lib/logger.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ const { isFunction, isNil, isString } = require('lodash');
 class Logger {
   constructor(config) {
     const {
-      log: { debug, warn, error, deprecate, inspectionDepth } = {},
+      log: { debug, warn, error, deprecate, inspectionDepth, enableColors } = {},
     } = config;
     this._inspectionDepth = inspectionDepth || 5;
+    this._enableColors = isNil(enableColors) ? process.stdout.isTTY : enableColors;
```
Comment (kibertoad) [inline lib/logger.js]: is `process.stdout.isTTY` available in browser? does this need defensive access chaining?

**Comment by kibertoad** [comment]
@lorefnon

**Comment by lorefnon** [comment]
@kibertoad Thanks for taking care of this. 

I have been travelling over the vacations and didn't get a chance to look into the above.

**Comment by kibertoad** [comment]
np, thanl you for the fix itself!

**Pull request merged**